### ISSUE TITLE
Add mini chartbot to the webpage.

### ISSUE THREAD

**Issue by ruchikakengal** [Mohit5Upadhyay/WeatherApplication#64]
i whould like to work on this.

**Comment by Mohit5Upadhyay** [comment]
Yes , @ruchikakengal  go ahead to implement mini chatbot functionality , but it will be dynamic to users interaction.

**Comment by PulkitShubham** [comment]
I would like to make a floating chatbot which would enhance the overall appearance.Kindly assign it to me

**Comment by ruchikakengal** [comment]
merged score is not visible in leaderboard.@Mohit5Upadhyay

**Comment by ruchikakengal** [comment]
Add hacktoberfestaccepted label to this PR @Mohit5Upadhyay
